### PR TITLE
PodmanPy Pod Operations

### PR DESCRIPTION
**Pull request by jwhonce**
* Pod operations and classes modeled after the Container operations
* unittests provided
* pylint coverage

Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: podman/domain/containers.py [hunk 1]
```diff
@@ -466,8 +463,8 @@ def top(self, **kwargs) -> Dict[str, Any]:
             ps_args (str): Optional arguments passed to ps
         """
         params = {
-            "ps_args": kwargs.get("ps_args", None),
-            "stream": kwargs.get("stream", None),
+            "ps_args": kwargs.get("ps_args"),
+            "stream": False,
```
Comment (TomSweeneyRedHat) [inline podman/domain/containers.py:467]: Is this right to set this to false all the time?

**Comment by TomSweeneyRedHat** [comment]
LGTM
one question for consideration.

File: podman/domain/containers.py (hunk 1 above)
Comment (mwhahaha) [reply podman/domain/containers.py:467]: IMHO yes because stream requires special handling.

**Review by mwhahaha — APPROVED** [review]

**Pull request merged**